es-bs-base/scss/vue/components/_verification-code.scss

```diff
@@ -0,0 +1,23 @@
+@use "../../variables" as variables;
+@use "../../mixins/breakpoints";
+
+.code-holder {
```

Comment:
Nit: These class names could probably be adjusted to slightly more specific, maybe by adding `verification-` in front?